### DISCUSSION TITLE
检测无其他maa功能异常

### DISCUSSION THREAD

**Discussion by 820512356** [MaaAssistantArknights/MaaAssistantArknights#10889] [Q&A] [answered]
版本Core Version: v5.8.0-beta.2，在22日更新后，双开maa提示无法在同一路径下运行，在我复制maa到另一个路径下后同时启动
在另一个maa尚未结束的时候，第一个maa提示我任务完成即将关机，我确定我设置的是正确的

**Comment by ABA2396** [comment, marked as answer]
这个检测是根据进程名拿的，如果权限不够会获取不到。确定两个权限等级一样吗
